src/command-helpers/scaffold.js

```diff
@@ -59,8 +79,54 @@ const writeScaffold = async (scaffold, directory, spinner) => {
   await writeScaffoldDirectory(scaffold, directory, spinner)
 }
 
+const writeABI = async (abi, contractName, abiPath) => {
+  let data = prettier.format(JSON.stringify(abi.data), {
+    parser: 'json',
+  })
+  let filePath = abiPath ? abiPath : `./abis/${contractName}.json`
```

Comment:
Maybe this could go into the function signature (or just turn the ternary into an `||`)? 🤔 

```javascript
const writeABI = async (abi, contractName, abiPath = `./abis/${contractName}.json`) => {
```